neo/app/assets/javascripts/lib/dc.leaflet.js

```diff
@@ -0,0 +1,434 @@
+dc.leafletChart = function(_chart) {
```

Comment:
Actually this is a library but not in webJar, so I put it under the /lib folder. If we want more interaction between leaflet and dc, we need to further extend this library